src/main/webapp/WEB-INF/views/addDriverToCar.jsp

```diff
@@ -0,0 +1,48 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Add driver</title>
+</head>
+<body>
+<h3>Add driver to car</h3>
+<form method="post" name="carDriverId">
+  Car ID <input type="text" name="carId"><br>
+  Driver ID <input type="text" name="driverId"><br>
+    <button type="submit">submit</button>
+</form>
+
+<h3>List of drivers:</h3>
+<table>
+    <tr>
+        <td>Id</td>
+        <td>Name</td>
+        <td>License number</td>
+    </tr>
+    <c:forEach items="${drivers}" var="driver">
+        <tr>
+            <td><c:out value="${driver.getId()}" /></td>
+            <td><c:out value="${driver.getName()}" /></td>
+            <td><c:out value="${driver.getLicenseNumber()}" /></td>
```

Comment:
I've discovered that this thing is equals to this *optionally
```suggestion
            <td><c:out value="${driver.id}" /></td>
            <td><c:out value="${driver.name}" /></td>
            <td><c:out value="${driver.licenseNumber}" /></td>
```

Reply:
agree with @X3NON-135